### PR TITLE
Update cutadapt to 1.9.1 for foss-2015b

### PR DESCRIPTION
**Pull request by verdurin**
Update homepage/description and authors.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/cutadapt/cutadapt-1.9.1-foss-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+pyshortver = '.'.join(pyver.split('.')[:2])
+
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+]
+
+sanity_check_paths = {
+    'files': ['bin/cutadapt', 'lib/python%s/site-packages/cutadapt/_align.so' % pyshortver],
```
Comment (boegel) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.9.1-foss-2015b-Python-2.7.10.eb]: @verdurin: please don't hardcode `.so`, use `SHLIB_EXT`:

```python
files': ['bin/cutadapt', 'lib/python%s/site-packages/cutadapt/_align.%s' % (pyshortver, SHLIB_EXT)],
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f1538c520be310cc01cc for a full test report.

File: easybuild/easyconfigs/c/cutadapt/cutadapt-1.9.1-foss-2015b-Python-2.7.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.9.1-foss-2015b-Python-2.7.10.eb]: cfr. https://github.com/hpcugent/easybuild-easyconfigs/pull/2298

Comment (verdurin) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.9.1-foss-2015b-Python-2.7.10.eb]: I just inherited that from the earlier version...

Will fix it later today.

Comment (boegel) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.9.1-foss-2015b-Python-2.7.10.eb]: A :cookie: if you also fix it in the other cutadapt easyconfigs! :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/359eaa8668d56dcc53ef for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**